server/rest/src/test/java/org/infinispan/rest/RestOperationsOffHeapTest.java

```diff
@@ -13,6 +13,8 @@
    public ConfigurationBuilder getDefaultCacheBuilder() {
       ConfigurationBuilder configurationBuilder = new ConfigurationBuilder();
       configurationBuilder.memory().storageType(StorageType.OFF_HEAP);
+      // Off heap and compatibility don't work together
+      configurationBuilder.compatibility().disable();
```

Comment:
Also this didn't work (test still fails), have to look as to why.

Reply:
Should be fixed as I made the super class not always enable compatibility.

Reply:
Aren't there any test methods that require compatibility?

Reply:
Sure they are at https://github.com/infinispan/infinispan/pull/5590/files#diff-56432ab419df420236bf86f0e4cea135R19

Off heap and compatibility is not a valid configuration together.

Reply:
I was assuming that the base class enabled compatibility because some of its test methods actually needed compatibility, but if not that's fine.